metric_providers/psu/energy/ac/xgboost/machine/provider.py

```diff
@@ -13,7 +13,7 @@
 from metric_providers.base import BaseMetricProvider
 
 class PsuEnergyAcXgboostMachineProvider(BaseMetricProvider):
-    def __init__(self, resolution):
+    def __init__(self, resolution, **_):
```

Comment:
Why did you choose to not implement the same change of signature with actual positional arguments as in the SDIAProvider?

Reply:
Good point :) no easy wins here.